scan-irb/report-zh-Hant.tex

```diff
@@ -73,143 +79,127 @@
-test result, you should contact a doctor or healthcare provider.}
+不管您的檢測結果如何，如果您感覺不舒服，您應該待在家裡，避免和其他人在一起。
+\textbf{如果您一直有症狀，您病得很重或擔心您自己的疾病，或對您的研究檢測結果有疑問，
+您應該聯繫醫生或醫療保健提供者。}
 
 \begin{itemize}
 
 
 \item
 
-  \textbf{If your test says ``Not Detected (Negative)''}, it means our lab did not find the
-  virus that causes COVID-19 in the sample that you sent to us. No test is
-  perfect, and there is a small chance that you could have the virus even though
-  your test is negative.
+  \textbf{若您的檢測結果顯示「未檢出（陰性）」}，即表示我們實驗室
+  未在您寄送給我們的樣本中發現引起 COVID-19 的病毒。沒有什麼檢測是完美的，
+  即使您的檢測結果為陰性，您也有些許可能性感染病毒。
 
 \item
 
-  \textbf{If your test says ``Detected (Positive)''}, it means our
-  lab found the virus that causes COVID-19 in the sample that you sent us. You
-  should stay home and avoid being around other people, especially if you are
-  feeling sick.
+  \textbf{若您的檢測結果顯示「檢出（陽性）」}，即表示我們實驗室
+  在您寄送給我們的樣本中發現引起 COVID-19 的病毒。您應該待在家中，避免接觸其他人，
+  尤其是您感到不舒服的時候。
 
-  Your result will be shared with the Washington State and local health
-  departments. You do \textbf{not} need to report these results yourself. The
-  health department may call you to ask questions about your illness and people
-  you may have been around while you were sick.
+  您的檢測結果將與華盛頓州和當地衛生部門共享。您不需要自己報告結果。衛生部門可能會打電話給您，
+  詢問您的病情以及您在生病時所接觸人員的情況。
 
 \item
 
-  \textbf{If your test says ``Inconclusive''}, it means our lab is not certain
-  of your test result. No test is perfect, and an inconclusive result happens
-  when we find a small amount of SARS-CoV-2 in your sample, but not enough for
-  us to say that you have COVID-19.
+  \textbf{若您的檢測結果顯示「不確定」}，即表明我們實驗室對您的檢測結果沒有把握。
+  沒有什麼檢測是完美的，當我們在您的樣本中發現少量的 SARS-CoV-2，但數量不足以讓我們說您患有
+  COVID-19 時，就會出現不確定的結果。
 
 \item
 
-  \textbf{If your test says ``Unable to be tested''}, it means there was either
-  not enough material on your sample for us to test, or there was an error when
-  we ran the test. Since we are not able to test your sample, we could not find
-  whether or not you have the virus that causes COVID-19.
+  \textbf{若您的檢測結果顯示「無法進行檢測」}，即表示 您的樣本中沒有足夠的物質供我們檢測，
+  或者我們進行檢測時出現錯誤。由於我們無法檢測您的樣本，所以無法發現您是否攜 帶導致 COVID-19
+  的病毒。
 
 \end{itemize}
 
 \bigskip
 
-\large \underline{\textbf{Resources}}
+\large \underline{\textbf{資源}}
 
-If you would like more information on COVID-19, please review these resources
-from public health partners:
+如果您想瞭解更多關於 COVID-19 的資訊，請查看公共衛生合作夥伴提供的這些資源:
 
 \begin{itemize}
 \item
-  Public Health Seattle and King County\\
-  \url{http://www.kingcounty.gov/covid}
+  Public Health \textemdash Seattle \& King County（西雅圖和金縣公共衛生）\\
+  \url{https://www.kingcounty.gov/depts/health/covid-19/languages/chinese.aspx}
 
 \item
 
-  Washington State Department of Health\\
-  \url{http://www.doh.wa.gov/Emergencies/Coronavirus}
+  Washington State Department of Health（華盛頓州衛生部）\\
+  \url{www.doh.wa.gov/Emergencies/NovelCoronavirusOutbreak2020COVID19/ChineseSimplified}
```

Comment:
Regarding this long URL: I'm not sure why line wrapping works automatically for the longer, following URL but not for this one. e.g. 
![image](https://user-images.githubusercontent.com/11384464/85345182-a8ee9400-b4a6-11ea-95d5-c93c961a9b91.png)

Reply:
I resolved this issue here and in https://github.com/seattleflu/lab-result-reports/pull/16 with `/sloppy`